stdlib/public/TensorFlow/Utilities.swift

```diff
@@ -279,6 +279,11 @@ public struct RandomNumberSequence : Sequence {
 
 /// This is a generic host-only op that hides the details of its impl in the SIL
 /// code. This makes reading/writing SIL based compiler unit tests simple.
+@inline(never)
+public func _hostOp<T>(_ x: T) {
+  print(x)
+}
+
 @inline(never)
 public func _hostOp<Scalar>(_ x: Tensor<Scalar>) {
```

Comment:
pls add a brief comment on why we are keeping this (the description in PR looks great!)

Reply:
Just noticed the pr was merged -- we can add this some time later. no rush.